### PR TITLE
[#363] Add test for CharacterCountSpec.js

### PR DESCRIPTION
**Pull request by jnamster**
Added test for CharacterCountSpec.js.

### CODE REVIEW

File: spec/javascripts/CharacterCountSpec.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    });
+
+    describe('noCKEditor', function() {
+      describe('home page', function() {
+        beforeEach(function() {
+          $('body').addClass('pages home')
+          loadFixtures('character_count.html');
+
+          onReadyCharacterCount();
+          $('#moment_why').keyup()
```
Comment (julianguyen) [inline spec/javascripts/CharacterCountSpec.js]: Add missing semicolon! Though semi-colons are necessary for JS, let's stick with ES6 linting conventions :)
Please update this throughout this document :D

File: spec/javascripts/CharacterCountSpec.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+          var editor = $('#moment_why');
+          editor.val(val)
+          noCKEditor(editor);
+        }
+
+        it("submit button should not be disabled", function() {
+          charLimit('aaaaaaa')
+          var input = $('input[type="submit"]');
+
+          expect(input.attr('disabled')).toBeFalsy()
```
Comment (julianguyen) [inline spec/javascripts/CharacterCountSpec.js]: Why not use toBeTrue() here instead? What was the reasoning to use falsy/truthy?

**Comment by julianguyen** [comment]
Please update this PR :D

File: spec/javascripts/CharacterCountSpec.js (hunk 1 above)
Comment (jnamster) [inline spec/javascripts/CharacterCountSpec.js]: Noted. semi-colon added.

File: spec/javascripts/CharacterCountSpec.js [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+          expect(input.attr('disabled')).toBeTrue();
+        });
+
+        it("submit button should not be disabled", function(); {
+          var overLimit = new Array(2002).join('a');
+          charLimit(overLimit);
+          var input = $('input[type="submit"]');
+
+          expect(input.attr('disabled')).toBeTruthy();
```
Comment (julianguyen) [inline spec/javascripts/CharacterCountSpec.js]: This should also be updated to `toBeTrue`

Comment (jnamster) [inline spec/javascripts/CharacterCountSpec.js]: Noted. Change added.

**Pull request merged**